v1/ast/parser_test.go

```diff
@@ -1245,10 +1245,10 @@ func TestFutureImports(t *testing.T) {
 	assertParseErrorContains(t, "keyword import + alias", "import future.keywords.in as xyz", "`future` imports cannot be aliased")
 
 	assertParseImport(t, "import kw with kw in options",
-		"import future.keywords.in", &Import{Path: RefTerm(VarTerm("future"), keywordsTerm, StringTerm("in"))},
+		"import future.keywords.in", &Import{Path: RefTerm(VarTerm("future"), InternedStringTerm("keywords"), StringTerm("in"))},
```

Comment:
"in" isn't interned?

Reply:
I only changed it here because `keywordsTerm` was a dependency of the test that I had removed :) We can intern `"in"`, but it was not one of the words that came up frequently when I put a println in `ast.StringTerm`.